app/namespace.go

```diff
@@ -580,6 +581,11 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				codecEndpointFlag,
 				codecPassAccessTokenFlag,
 				codecIncludeCredentialsFlag,
+				&cli.StringFlag{
+					Name:    cloudProviderFlagName,
+					Usage:   `Cloud provider for the namespace to be created for, currently support [aws, gcp].  For this version, if not specified, we default to aws`,
+					Aliases: []string{"cp"},
```

Comment:
You could add a defaultValue in cli.StringFlag so you avoid the check between line 610 and 612

Reply:
Yes, that's the "follow up" change i mentioned in the comment.  If i do this, it will be a breaking change, and some existing customer may get affected.  So would want a release of this version to let customer know about the flag, and for next release, we will enforce the flag

Reply:
i mean you could add a `defaultValue`  as to "aws" and that will automatic apply if user doesn't given an input.

Reply:
oh, the following line "Value: CloudProviderAWS," is the default value